0-generate-pkg-list

```diff
@@ -25,7 +25,11 @@ detect_rescue_file
 # Promote osg-version to testing #
 ##################################
 promote_osg_version() {
-    for ver in ${versions[@]/upcoming/}; do # We don't build osg-version for upcoming
+    for ver in ${versions[@]}; do
+        case $ver in
+            # We don't build osg-version for upcoming or 3.5+
+            3.5 | upcoming ) continue ;;
```

Comment:
I believe `$ver` is the full version string here, e.g. `3.4.34`. Does this actually work for that?

Reply:
herm... let's see...

Reply:
No, you're right it won't work - i was mistaken about the format of `$ver` - i'll fix.

Reply:
Globs are allowed in the `case` patterns, so I check for `3.5.*` instead.